### PR TITLE
adjust plugins

### LINKED ISSUE #183: 调整默认插件
- [ ] 默认开启 static
- [ ] 移除 cors
- [ ] 移除 rest

### PR DESCRIPTION
**Pull request by dead-horse**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] commit message follows commit guidelines

##### Description of change

- 移除 cors 和 rest
- 默认开启 static，static 现在版本懒加载，无副作用

closes #183

### CODE REVIEW

**Comment by atian25** [comment]
i18n 要内置么?

**Comment by fengmk2** [comment]
测试等 https://github.com/eggjs/egg-onerror/pull/10 就会修复。

File: config/plugin.js [hunk 1]
```diff
@@ -134,38 +134,14 @@ module.exports = {
...
-
-  /**
-   * CORS
-   * @member {Object} Plugin#cors
-   * @property {Boolean} enable - `false` by default
-   * @since 1.0.0
-   */
-  cors: {
-    enable: false,
-    package: 'egg-cors',
```
Comment (fengmk2) [inline config/plugin.js]: @dead-horse package.json 里面也删除了。

Comment (dead-horse) [inline config/plugin.js]: fixed

**Pull request merged**